src/nfs/nfsv4.rs

```diff
@@ -60,6 +60,19 @@ pub fn number_of_clients() -> i64 {
     0
 }
 
+pub fn number_of_exports() -> i64 {
+    let etab = VAR_NFSDV4.to_owned() + "/etab";
+    if path_exists(&etab) {
+        let open_etab = File::open(etab).expect("file not found");
```

Comment:
I think it could fail for other reason than `file not found`. Maybe you should just `unwrap` it to get the real error message.

Reply:
The .unwrap() panics, however the next line 67 will also panic if the open_etab reaches the expect(). I think the issue here is not with the patch itself, but with how I handle errors and it needs to be totally reworked. A good example is what you pointed out, I cared about opening the file and setting an error to avoid panic with unwrap(), but I do nothing with that error and it will panic with BufReader.

Would be great if we get a patch with these changes, these bad error handle should not be hard to be spotted and fixed. Something I might work on in a couple of weeks if nobody else sends a patch. But yeah, good catch @nlewo.

Reply:
I will merge this patch as it is, and I will open an ISSUE about the error handlers that need to be reworked. I will point out this PR to keep the conversation alive. Thanks again guys for the contribution.

Reply:
I'm not sure to understand your answer since both `expect` and `unwrap` panics on `Err`. 

By reading the doc, i actually realized `expect` also outputs the error message. So, my previous message was not completely true. I however still think the `file not found` message can be wrong (on a permission error for instance). 
That's however not really important and i'm fine with the merge.

Also I agree with instead of panic we should return the error but as you said, that's another topic :wink: